### PR TITLE
airshare: update python resources

### PR DESCRIPTION
**Pull request by alex**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

resolves a GHSA in certifi

### CODE REVIEW

File: Formula/airshare.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ class Airshare < Formula
   url "https://files.pythonhosted.org/packages/cb/a2/d59c18cd6a143bf860c29acb70552b7351fd7e0f56213be86b624601106b/Airshare-0.1.6.tar.gz"
   sha256 "138976933bcf2c0bc99f18901242544919e9fafe43735b4f83b19d4ae5964be7"
   license "MIT"
-  revision 3
+  revision 4
```
Comment (chenrui333) [inline Formula/airshare.rb:9]: we need to revert this change, this would just be a rebuild

**Review by chenrui333 — APPROVED** [review]

**Comment by alex** [comment]
@chenrui333 thanks for handling all of these (I sent them and then went to work so I wasn't able to reply quickly to the feedback!)
